### PR TITLE
[Please Merge] Php kubernetes database

### PR DESCRIPTION
**Pull request by KugliLuca**
The like-service will now create a Database "likeDb" on the MariaDB service along with a table "like" in it.

### CODE REVIEW

File: src/like-service/app/Console/Commands/CreateDatabase.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    {
+        $host = getenv('UNGUARD_MARIADB_SERVICE_HOST', false);
+        $port = getenv('UNGUARD_MARIADB_SERVICE_PORT_MYSQL', false);
+        $rootuser = 'root';
+        $password = getenv('MARIADB_PASSWORD', false);
+        $address = $host . ":" . $port;
+
+        $connection = new PDO("mysql:host=$address", $rootuser, $password);
+        try {
+            $connection->exec('CREATE DATABASE likeDb');
```
Comment (Dirli-V) [inline src/like-service/app/Console/Commands/CreateDatabase.php]: [DISCUSS] I think we should use the DB_DATABASE environment variable for the database name

File: src/like-service/app/Console/Commands/CreateDatabase.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        } catch (PDOException $e) {
+            Log::notice('PDO-Exception, probably due to database "likeDb" already existing');
```
Comment (Dirli-V) [inline src/like-service/app/Console/Commands/CreateDatabase.php]: [ISSUE] Please add the exception as context
```suggestion
            Log::notice('PDO-Exception, probably due to database "likeDb" already existing', [$e]);
```

File: src/like-service/database/migrations/2023_07_13_073935_create_like_table.php [hunk 3]
```diff
@@ -13,10 +13,10 @@ class CreatePasswordResetsTable extends Migration
      */
     public function up()
     {
-        Schema::create('password_resets', function (Blueprint $table) {
-            $table->string('email')->index();
-            $table->string('token');
-            $table->timestamp('created_at')->nullable();
+        Schema::create('like', function (Blueprint $table) {
+            $table->integer('postId', false);
+            $table->integer('UserId', false);
```
Comment (Dirli-V) [inline src/like-service/database/migrations/2023_07_13_073935_create_like_table.php]: [NIT] postId's first letter is lower case vs UserId's letter is upper case. Please unify them

**Review by Dirli-V — CHANGES_REQUESTED** [review]

File: src/like-service/database/migrations/2023_07_13_073935_create_like_table.php (hunk 3 above)
Comment (KugliLuca) [reply src/like-service/database/migrations/2023_07_13_073935_create_like_table.php]: Changed to lower case

File: src/like-service/app/Console/Commands/CreateDatabase.php (hunk 2 above)
Comment (KugliLuca) [reply src/like-service/app/Console/Commands/CreateDatabase.php]: Added

File: src/like-service/app/Console/Commands/CreateDatabase.php (hunk 1 above)
Comment (KugliLuca) [reply src/like-service/app/Console/Commands/CreateDatabase.php]: Changed

**Review by Dirli-V — APPROVED** [review]

**Pull request merged**